hypercall/include/x64/mv_reg_t.h

```diff
@@ -194,8 +194,202 @@ enum mv_reg_t
         mv_reg_t_cr8 = 69,
         /** @brief defines the xcr0 register (Intel Only) */
         mv_reg_t_xcr0 = 70,
+        /** @brief defines the es_type register */
+        mv_reg_t_es_type = 71,
```

Comment:
You cannot add more of these. mv_reg_t is dictated by the MicroV. If you attempted to use these, MicroV would just yell stating that it has no idea what those registers are. 

What you are trying to define here are the individual bit fields of the segment attributes. If you cannot simply copy from mv_reg_t_xx_attrib to the KVM struct, you will have to parse the results of mv_reg_t_xx_attrib for each field that KVM is asking for.

Reply:
No in other words, remove all of these, and instead, ask MicroV for mv_reg_t_xx_attrib  (replace xx with the selector that you are trying to get), and then either copy if the struct layout is the same as what MicroV is returning (it likely is as, or is very close), or parse the bit fields and copy each bit field by hand.